samples/sampler/stories/default/Scroller.js

```diff
@@ -0,0 +1,72 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import React from 'react';
+import ri from '@enact/ui/resolution';
+import {ScrollerBasic as UiScrollerBasic} from '@enact/ui/Scroller';
+import {storiesOf} from '@storybook/react';
+
+import BodyText from '@enact/agate/BodyText';
+import Scroller from '@enact/agate/Scroller';
+
+const prop = {
+	direction: ['both', 'horizontal', 'vertical'],
+	focusableScrollbarOption: {
+		false: false,
+		true: true,
+		'&quot;byEnter&quot;': 'byEnter'
+	},
+	scrollbarOption: ['auto', 'hidden', 'visible'],
+	scrollModeOption: ['native', 'translate']
+};
+
+const ScrollerConfig = mergeComponentMetadata('Scroller', UiScrollerBasic, Scroller);
+
+storiesOf('Agate', module)
+	.add(
+		'Scroller',
+		() => (
+			<Scroller
+				style={{height: ri.scaleToRem(540), width: ri.scaleToRem(1540)}}
```

Comment:
Why it has a fixed size?

Reply:
Let's remove `:not([style])` from `ThemeEnvironment.module.less` and remove styles from Scroller and VL.